### DISCUSSION TITLE
Validate JWT Signature

### DISCUSSION THREAD

**Discussion by dcaponi** [pocketbase/pocketbase#3085] [Q&A] [answered]
Hey there.

Im building a system that uses pocketbase for auth and after completing the login flow on a sveltekit app, I have the pb_auth token JWT on a cookie. I want my frontend to send requests with said cookie to a python backend to do some work (most of the libs I need are in python so this was the easiest approach for me).

As it stands, I can only find literature for validating the JWT by making a call to pocketbase with the token and getting a yes/no. I'd like to avoid an extra http call if possible. Normally I'd share the signing secret among backends and validate the signature using some JWT validation library.

Is standalone JWT signature validation with a shared secret not a thing here or do I have to make the extra HTTP call every time?

**Comment by ganigeorgiev** [comment, marked as answer]
You have to make the extra HTTP call every time. Each auth record has their own unique JWT secret key (see https://github.com/pocketbase/pocketbase/issues/144).
